perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -60,9 +60,10 @@ olcAttributeTypes: {55}( 1.3.6.1.4.1.8057.2.80.61 NAME 'isAssignedWithSubgroups'
 olcAttributeTypes: {56}( 1.3.6.1.4.1.8057.2.80.62 NAME 'tcsMails' DESC 'List of users mail addresses used for TCS' EQUALITY caseIgnoreIA5Match SUBSTR caseIgnoreIA5SubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.26{256} )
 olcAttributeTypes: {57}( 1.3.6.1.4.1.8057.2.80.63 NAME 'sshPublicKey' DESC 'User`s public SSH key.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {58}( 1.3.6.1.4.1.8057.2.80.64 NAME 'sponsoredMembershipInOrganizations' DESC 'List of sponsored memberships in organizations.' EQUALITY caseExactMatch SUBSTR caseExactSubstringsMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {19}( 1.3.6.1.4.1.8057.2.80.23 NAME 'schacPersonalUniqueCodes' DESC 'Unique codes of the user in Perun' EQUALITY caseIgnoreMatch SUBSTR caseIgnoreSubstringsMatch ORDERING caseIgnoreOrderingMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
Attribute value ordering should be sequential ``{19}`` -> ``{59}`` unless you want to re-order them. But we have another pull-request with added attribute, so it should be probably 60 now.
OID should be also sequentially assigned from CESNET range. Hence ``1.3.6.1.4.1.8057.2.80.65`` or ``....66`` (as we added another attribute).

Reply:
copy-paste mistake. Fixed

Reply:
Fixed